steam/webauth.py

```diff
@@ -224,6 +224,8 @@ def login(self, password='', captcha='', email_code='', twofactor_code='', langu
                 raise EmailCodeRequired(resp['message'])
             elif resp.get('requires_twofactor', False):
                 raise TwoFactorCodeRequired(resp['message'])
+            elif 'short time period' in resp.get('message', ''):
+                raise TooManyTimesFailed(resp['message'])
```

Comment:
Can we rename `TooManyTimesFailed` to `TooManyLoginFailures`?

Reply:
yeah, that is better naming.